### PR TITLE
Refine Welcome Tab content

### PR DESCRIPTION
**Pull request by 0aveRyan**
Changes to content strings and add currentBrand display name.

### CODE REVIEW

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json [hunk 1]
```diff
@@ -4,21 +4,23 @@
             "name": "tab1",
             "title": "YOUR CONTENT",
             "subtitle": "Publish boldly with WordPress Blocks.",
-            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate layout assembly and storytelling with premade designs.",
+            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate telling your story or tending your store with professional designs.",
```
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json:7]: Is the second **store** use intentional? Or should that be switched with **site** or **website**?

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json [hunk 2]
```diff
@@ -4,21 +4,23 @@
...
+            "buttonText": "Try",
+            "buttonUrl": "",
```
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json:9]: Do we need this?

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json (hunk 1 above)
Comment (0aveRyan) [reply src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json:7]: This one was intentional to exclude from replacements, but thanks for checking!

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json (hunk 2 above)
Comment (0aveRyan) [reply src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json]: Ah, didn't mean this one to go up, I was moving quick through history. Tried some call-to-action button treatments but it was too cluttered.

**Review by officiallygod — APPROVED** [review]

**Comment by abhijitb** [comment]
QA done.

[welcome-step-content-refinement.webm](https://user-images.githubusercontent.com/533330/215762637-f943f5ae-9921-486f-bce2-3550b88917b6.webm)

**Pull request merged**